content/docs/ops/maintenance-list.md

```diff
@@ -47,11 +47,27 @@ If you see a way to make this checklist better, just submit a PR to the
 
 ## Ensure all VMs are running the current stemcell
 
-- Get the latest stemcell version from http://bosh.cloudfoundry.org/stemcells/
-  for AWS Xen-HVM Light
+- Check the latest stemcell version for AWS Xen-HVM Light at http://bosh.cloudfoundry.org/stemcells/
 
-- From the jumpbox in each of our four environments, run `bosh deployments` and verify the stemcell in-use
-for each deployment is current.
+- From the jumpbox in each of our four environments, `tooling`, `development`, `staging` and `production`, run `bosh deployments` and verify the stemcell in use for each deployment is current. For example, the 3431.13 stemcells are outdated below:
+```sh
+root@PRODUCTION:/tmp/build/8e72821d$ bosh deployments | grep go_agent
+admin-ui             	clamav/9                           	bosh-aws-xen-hvm-ubuntu-trusty-go_agent/3445   	-	outdated
+cf-production        	clamav/9                           	bosh-aws-xen-hvm-ubuntu-trusty-go_agent/3431.13	-	none
+                     	awslogs/27                         	bosh-aws-xen-hvm-ubuntu-trusty-go_agent/3445
+cf-production-diego  	clamav/9                           	bosh-aws-xen-hvm-ubuntu-trusty-go_agent/3431.13	-	none
+concourse            	clamav/9                           	bosh-aws-xen-hvm-ubuntu-trusty-go_agent/3445   	-	latest
+kubernetes           	cron/17                            	bosh-aws-xen-hvm-ubuntu-trusty-go_agent/3445   	-	latest
+logsearch            	cron/17                            	bosh-aws-xen-hvm-ubuntu-trusty-go_agent/3445   	-	latest
+shibboleth-production	clamav/9                           	bosh-aws-xen-hvm-ubuntu-trusty-go_agent/3445   	-	latest
+```
+
+- When the stemcells are out-of-date:
+  - Review the release notes at http://bosh.cloudfoundry.org/stemcells/bosh-aws-xen-hvm-ubuntu-trusty-go_agent
+  - Trigger the appropriate `deploy-...` jobs in Concourse
+    - Triggering more than 3 jobs simultaneously is not advised
+  - Trigger the `cf-production` jobs in the `deploy-cf` pipeline only after development and staging have completed
```

Comment:
We don't use `passed` constraints linking development to staging and production, so it doesn't matter if development has already deployed the stemcell. And it technically won't hurt to deploy production before staging is done, since the production deploy won't grab a stemcell version that hasn't passed staging already. Also, this pattern isn't specific to the cf deployment--it applies to basically all of our bosh deploy pipelines.

Reply:
Ok, what I observed then was likely coincidence. I'll strike that.

Reply:
Actually, I could use some guidance/clarification if anyone can help.

For 3445, what I see is that deploy-cf-new/jobs/deploy-diego-staging/builds/72 finished at 2017-08-15 12:54:
```
150020  deploy-cf-new/deploy-diego-staging  72     succeeded  2017-08-15@11:54:59-0700  2017-08-15@12:54:07-0700  59m8s
```

Then the smoke tests passed, but there are so many that they've rolled off.

At that point I noticed that `acceptance-tests-staging` had not run, so I manually started it  From there the rest of the production release happened.

How was the release supposed to happen? I thought the deploy of cf to production was something that we manually gated. Is that gate `acceptance-tests-staging` or is it something else?

Reply:
The authoritative place to check the rules is the [pipeline config](https://github.com/18F/cg-deploy-cf/blob/master/pipeline.yml). But looking at this again, I'm not sure that it's useful to refer to specific pipelines--can we just drop references to `deploy-cf` here and leave it at "Trigger the appropriate jobs in concourse"?